### PR TITLE
REST APi accepts new project param that can be used to pass devops pr…

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Automatic review needs to fetch artifact from internal project. Currently REST API always query for download url using public project name. This PR is to support additional parameters so REST API caller can pass which project to be used to download artifact from pipeline.

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs [hunk 1]
```diff
@@ -88,7 +88,8 @@ IConfiguration configuration
             string codeFileName = null,
             string baselineCodeFileName = null,
             bool commentOnPR = true,
-            string language = null)
+            string language = null,
+            string projet = "public")
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:92]:
```suggestion
            string project = "public")

```

File: src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs [hunk 2]
```diff
@@ -114,7 +115,7 @@ IConfiguration configuration
                 
                 using var memoryStream = new MemoryStream();
                 using var baselineStream = new MemoryStream();
-                var codeFile = await _reviewManager.GetCodeFile(repoName, buildId, artifactName, packageName, originalFileName, codeFileName, memoryStream, baselineCodeFileName: baselineCodeFileName, baselineStream: baselineStream);
+                var codeFile = await _reviewManager.GetCodeFile(repoName, buildId, artifactName, packageName, originalFileName, codeFileName, memoryStream, baselineCodeFileName: baselineCodeFileName, baselineStream: baselineStream, project: projet);
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs:118]:
```suggestion
                var codeFile = await _reviewManager.GetCodeFile(repoName, buildId, artifactName, packageName, originalFileName, codeFileName, memoryStream, baselineCodeFileName: baselineCodeFileName, baselineStream: baselineStream, project: project);

```

File: src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs [hunk 3]
```diff
@@ -586,20 +586,21 @@ public async Task UpdateReviewBackground()
             string codeFileName,
             MemoryStream originalFileStream,
             string baselineCodeFileName = "",
-            MemoryStream baselineStream = null
+            MemoryStream baselineStream = null,
+            string project = ""
```
Comment (weshaggard) [inline src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:590]: Do we want this to be empty here?

Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs:590]: Thanks. this should be defaulted to public. I will check and fix it.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**